cfme/tests/cloud/test_targeted_refresh.py

```diff
@@ -0,0 +1,201 @@
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [
+    test_requirements.ec2,
+    pytest.mark.ignore_stream('5.10', 'upstream')
+]
+
+
+@pytest.mark.manual
+def test_ec2_targeted_refresh_instance():
+    """
+    Polarion:
+        assignee: mmojzis
+        casecomponent: Cloud
+        initialEstimate: 1 1/6h
+        startsin: 5.9
+
+        testSteps:
```

Comment:
testSteps should be numbered and have expectedResult (as it's a manual test).
Examples can be found in other manual test cases.

Reply:
If you move tests, you can just move. I'd leave it in the description or add tests steps _and_ expected results if you want to make it better

Reply:
Actually testSteps == expectedResults for now. I create instance in testStep and then just check it's created. I would only duplicate test steps to expectedResults...